docs/admin/executors/deploy_executors_kubernetes.mdx

```diff
@@ -104,48 +85,14 @@ Environment.
 ### Batch Changes
 
 To run [Batch Changes](/batch_changes/) on
-Kubernetes, [Native Server-Side Batch Changes](/admin/executors/native_execution) must be enabled.
+Kubernetes, [native Server-Side Batch Changes](/admin/executors/native_execution) must be enabled.
+
+### Docker Image
+
+The Executor Docker image is available on Docker Hub at [`sourcegraph/executor-kubernetes`](https://hub.docker.com/r/sourcegraph/executor-kubernetes/tags).
 
 ### Example Configuration YAML
 
 See
 the [local development YAMLs](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/blob/cmd/executor/kubernetes)
-for an example of how to configure the Executor in Kubernetes.
-
-## Deployment
-
-Executors on Kubernetes require specific RBAC rules to be configured in order to operate correctly.
-See [RBAC Roles](#rbac-roles) for more information.
-
-### Step-by-step Guide
-
-Ensure you have the following tools installed:
-
-- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
-- [Helm](https://helm.sh/) if you're installing Sourcegraph with `helm`.
-
-#### Deployment via Kustomize
-
-Please refer to the [Sourcegraph Kustomize docs](/admin/deploy/kubernetes/kustomize) for the latest instructions.
-
-To include Native Kubernetes Executors, see [configure Sourcegraph with Kustomize](/admin/deploy/kubernetes/configure) on how to specify the component (`components/executors/k8s`).
-
-#### Deployment via Helm
-
-Please refer to the [Sourcegraph Helm docs](/admin/deploy/kubernetes/helm#quickstart) for the latest instructions.
-
-To specifically deploy Executors,
-1. Create an overrides file, `override.yaml`, with any other customizations you may require.
-    1. See [details on configurations](/admin/deploy/kubernetes/helm#configuration).
-2. Run the following command:
-    ```bash
-    helm upgrade --install --values ./override.yaml --version <your Sourcegraph Version> sg-executor sourcegraph/sourcegraph-executor-k8s
-    ```
-3. Confirm executors are working by checking the _Executors_ page under **Site admin > Executors > Instances** .
-
-## Note
-
-Executors deployed on Kubernetes do not use [Firecracker](/admin/executors/#how-it-works).
-
-If you have security concerns, consider deploying via [terraform](/admin/executors/deploy_executors_terraform)
-or [installing the binary](/admin/executors/deploy_executors_binary) directly.
```

Comment:
This info looks important - was it moved elsewhere?

Reply:
no, I replaced this section with my rewrite that is (in my opinion) less noise and more direct.

For example, I don't think there is any need to instruct readers to install kubectl / helm, as they presumably would have already done so to deploy their Sourcegraph instance using those tools

Reply:
However, I did retain the `Notes` section, which was renamed to `Additional Notes` and moved to the bottom of the page.

Reply:
That makes sense.